Suuports simbolic diff for leaky_relu

edvgha opened the pull request:
Summary: supports  symbolic differentiation for leaky_relu

Test Plan:
test/test_jit.py
test/test_ops.py

Differential Revision: D28458898

Krovatkin commented on torch/testing/_internal/common_methods_invocations.py:
```diff
@@ -5018,6 +5024,19 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
+           supports_out=False,
+           autodiff_nonfusible_nodes=["aten::leaky_relu"],
+            skips=(
+                SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=[torch.float32]),),),
```
what was the issue with this test?

edvgha replied:
in test_variant_consistency_jit function we have: check_alias_annotation where torch::jit::getOperatorAliasMap() get queried for leaky_relu, but leaky_relu is not part of that map:
https://github.com/pytorch/pytorch/blob/9def776cd6d775779db9839d942a01b4b082175c/torch/csrc/jit/passes/normalize_ops.cpp#L63

Krovatkin reviewed (approved):
:shipit:

edvgha committed:
Suuports simbolic diff for leaky_relu (#58337)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/58337

supports  symbolic differentiation for leaky_relu

Test Plan:
test/test_jit.py
test/test_ops.py

Differential Revision: D28458898

fbshipit-source-id: d9c877e3e0626266fbcfdf461b12a337992309cc
